### PR TITLE
Microsoft.dotNetFramework version 4.7.3081

### PR DESCRIPTION
**Pull request by benbou8231**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/18672)

### CODE REVIEW

File: manifests/m/Microsoft/dotNetFramework/4.7.3081/Microsoft.dotNetFramework.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotNetFramework
+PackageVersion: 4.7.3081
+PackageLocale: en-US
+Publisher: Microsoft
+PackageName: .NET Framework Developer Pack
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotNetFramework/4.7.3081/Microsoft.dotNetFramework.locale.en-US.yaml:8]:
```suggestion
PackageName: Microsoft .NET Framework SDK
```

File: manifests/m/Microsoft/dotNetFramework/4.7.3081/Microsoft.dotNetFramework.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotNetFramework
+PackageVersion: 4.7.3081
+PackageLocale: en-US
+Publisher: Microsoft
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotNetFramework/4.7.3081/Microsoft.dotNetFramework.locale.en-US.yaml:7]:
```suggestion
Publisher: Microsoft Corporation
```

File: manifests/m/Microsoft/dotNetFramework/4.7.3081/Microsoft.dotNetFramework.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotNetFramework
+PackageVersion: 4.7.3081
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotNetFramework/4.7.3081/Microsoft.dotNetFramework.installer.yaml:5]:
```suggestion
PackageVersion: 4.7.03081
```

File: manifests/m/Microsoft/dotNetFramework/4.7.3081/Microsoft.dotNetFramework.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotNetFramework
+PackageVersion: 4.7.3081
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotNetFramework/4.7.3081/Microsoft.dotNetFramework.yaml:5]:
```suggestion
PackageVersion: 4.7.03081
```

File: manifests/m/Microsoft/dotNetFramework/4.7.3081/Microsoft.dotNetFramework.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotNetFramework
+PackageVersion: 4.7.3081
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotNetFramework/4.7.3081/Microsoft.dotNetFramework.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.7.03081
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by benbou8231** [comment]
I just noticed that "wingetcreate submit" discarded the -DevPack suffix of the version number used in the folder name. I guess I'll have to create a new parent folder for the DevPacks, probably dotNetFrameworkDevPack.

**Comment by denelon** [comment]
@benbou8231 do you think that is a bug in the [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create)?

**Pull request closed without merging**